go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/prometheus/client_golang v1.0.0
 	github.com/satori/go.uuid v1.2.0
 	github.com/shirou/gopsutil v0.0.0-20190731134726-d80c43f9c984
-	github.com/sirupsen/logrus v1.4.0
+	github.com/sirupsen/logrus v1.4.0 // indirect
```

Comment:
What is the indirect?

Reply:
removed the whole line, it appears that logrus library is used by some other packages and since it was used also by elrond-go and was present in go.mod, goland did not remove it automatically.